src/mains/unix.cpp

```diff
@@ -525,6 +530,12 @@ void    ProcessConfigStr (_String& conf)
             exit (0);
         }
 
+        case 'v':
+        case 'V': {
+            fprintf(stderr, "\n%s\n", hy_version );
```

Comment:
I would just use

```
fprintf(stderr, "\n%s\n", GetVersionString()->getStr() );
```

that way you don't have to create the version string regradless of what the options are.